src/js/blocks/hero/block.js

```diff
@@ -330,14 +324,14 @@ registerBlockType( 'uri-cl/hero', {
 								label="Use WordPress caption"
 								help="Setting a custom caption below will override any WordPress caption."
 								checked={ attributes.use_caption }
-								onChange={ content => setAttributes( { use_caption: content } ) }
+								onChange={ ( content ) => setAttributes( { use_caption: content } ) }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'use_caption' is not in camel case.](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4811013)